eyeshade/workers/wallet.js

```diff
@@ -116,6 +116,8 @@ exports.initialize = async (debug, runtime) => {
 
 /*
```

Comment:
I think we should avoid committing WIP code in commented out form

Reply:
it's not WIP. it's what i use when debugging things. i'll take it out.